packages/react-dom/src/shared/DOMProperty.js

```diff
@@ -226,6 +221,25 @@ const properties = {};
   );
 });
 
+// defaultValue stringifies boolean values for backward compatability
+// from when it was treated as a reserved property. Reserved
+// properties skip most property validation.
+properties.defaultValue = new PropertyInfoRecord(
+  'defaultValue',
+  OVERLOADED_BOOLEAN,
+  false,
+  'value',
+  null,
+);
+
+properties.defaultChecked = new PropertyInfoRecord(
```

Comment:
But I do wonder about this one. Since defaultChecked was reserved, I believe there was actually a mismatch between `checked` and `defaultChecked`. `checked` is a `BOOLEAN` value, whereas (I believe) `defaultValue` was effectively an `OVERLOADED_BOOLEAN`.

I think they should be consistent, but just wanted to give a heads up that this could raise new warnings for consumers.